types/bull/index.d.ts

```diff
@@ -364,7 +364,7 @@ declare namespace Bull {
      * A boolean which, if true, removes the job when it successfully completes.
      * Default behavior is to keep the job in the completed set.
      */
-    removeOnComplete?: boolean;
+    removeOnComplete?: boolean | number;
```

Comment:
You should add to the docstring what a `number` represents.

Reply:
Honestly I'm not using Github that much. 
How am I supposed to amend my commit?

Reply:
But then how things will be merged ?
I went straight on to the file and dit the change in place ...

Reply:
Hmm, I dont quite understand. Anyways, its not a huge deal so I will approve.

Reply:
I meant, I just went to the file through github web interface, modified the file and it proposed me to add a PR .... done.  There must be a smarter way to add something to the PR than going through an entire new commit ? And in case I do/did, how do you replace this commit by this new commit ? Would the approval process would have to restart from scratch ... and so on ... all the questions that someone not fluent with github PR process would be asking to himself ...
Now this is a small modification and the author of bull is rewriting Bull entirely using Typescript.
Thanks for the approval  ...

Reply:
Ohhh, didn't realize you were using the Github web interface. I am just very used to people using the Git CLI. I would recommend getting familiar with it. The process there would be:

1. Pull down branch associated with this PR using Git CLI.
2. Modify file locally.
3. Create commit.
4. Push to this branch.